src/vector.js

```diff
@@ -346,18 +295,25 @@ class Vector {
 
       if (a.length === b.length && a.length === 3) {
         let params = [
-          a[1] * b[2] - a[2] * b[1],
-          a[2] * b[0] - a[0] * b[2],
-          a[0] * b[1] - a[1] * b[0]
+          Vector.vectorSub(
```

Comment:
Let's remove this

Reply:
Removal of this change will result in inaccurate calculations.